baseplate/experiments/__init__.py

```diff
@@ -183,15 +177,14 @@ def variant(self, name, user=None, bucketing_event_override=None,
 
         if do_log:
             assert bucketing_id
-            if self._event_logger:
-                self._event_logger.log(
-                    experiment=experiment,
-                    variant=variant,
-                    user_id=inputs.get('user_id'),
-                    logged_in=inputs.get('logged_in'),
-                    cookie_created_timestamp=inputs.get('cookie_created_timestamp'),
-                    app_name='r2'
-                )
+            self._event_logger.log(
+                experiment=experiment,
+                variant=variant,
+                user_id=inputs.get('user_id'),
+                logged_in=inputs.get('logged_in'),
+                cookie_created_timestamp=inputs.get('cookie_created_timestamp'),
+                app_name=inputs.get('app_name'),
```

Comment:
I think you also want to pass in `extra_event_fields` in some way, unless we want to get rid of that?